bot/exts/help_channels.py

```diff
@@ -28,17 +28,21 @@
 """
 
 AVAILABLE_MSG = f"""
-This help channel is now **available**, which means that you can claim it by simply typing your \
-question into it. Once claimed, the channel will move into the **Python Help: Occupied** category, \
-and will be yours until it has been inactive for {constants.HelpChannels.idle_minutes} minutes or \
-is closed manually with `!close`. When that happens, it will be set to **dormant** and moved into \
-the **Help: Dormant** category.
-
-Try to write the best question you can by providing a detailed description and telling us what \
-you've tried already. For more information on asking a good question, \
-check out our guide on **[asking good questions]({ASKING_GUIDE_URL})**.
+**Send your question here to claim the channel**
+This channel will be dedicated to answering your question only. We’ll try to answer and help you solve the issue.
+
+**Keep in mind:**
+• It's always ok to just ask your question. You don't need permission.
+• Explain what you expect to happen and what actually happens.
+• Include a code sample and error message, if you got one.
```

Comment:
Possible grammer edit needed?
`Include a code sample and/or error message, if you have either.`

Reply:
The intention was that "if you got one" would mean "if you got any of the two things. Maybe using "any" instead of "one" would be better, though. What do you think?

Reply:
This has been updated. I'm happy with how it looks now.

Reply:
I'm still not sure about the use of got here. Surely "have" is the better possessive, since they still have them.

I'll leave this to the core team to decide on though.

Reply:
The word "got" here refers to the things they got when they were trying it.